css/css-values/subresource-criticality/tentative/font-face-used.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>used @font-face blocks load</title>
+<link rel=help href="https://github.com/w3c/csswg-drafts/issues/1088">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<style>
+@font-face {
+  font-family: Revalia;
+  src: url("/fonts/Revalia.woff?pipe=trickle(d1)");
+}
+#foo {
+  font: 25px/1 Revalia;
+}
+</style>
+<div id=foo>XXX</div>
```

Comment:
In 2/4 font-using tests in this PR, the `id=foo` element comes before the `<style>` that includes the font, and in the other 2/4 it is the opposite. Can we make this consistent? (Not that it really matters much...)